gui/wxpython/tplot/frame.py

```diff
@@ -833,6 +833,7 @@ def OnRedraw(self, event=None):
                 except (ValueError, AttributeError):
                     GMessage(message=_("Incorrect coordinates format, should "
```

Comment:
Shouldn't this be GError like in the cases below? Otherwise seems good, although I haven't actually tested it.

Reply:
Yes you are right. Fixed in the commit [ab10998](https://github.com/OSGeo/grass/commit/ab1099898ff82e7c63e6db697eaab64eaf8ea81e).